scripts/migrate.cypher

```diff
@@ -16,3 +16,70 @@ WHERE sm.primaryId IS NULL
 SET sm.primaryId = sm.igoId
 REMOVE sm.igoId
 RETURN true;
+
+
+// SCHEMA VERSION: v2.0
+// ------------------------------------------------------------
+
+// request-level property name changes
+// - projectId --> igoProjectId
+// - requestId --> igoRequestId
+// - recipe --> genePanel
+MATCH (r: Request)
+WHERE r.igoProjectId IS NULL
+SET r.igoProjectId = r.projectId
+REMOVE r.projectId
+RETURN true;
+
+MATCH (r: Request)
+WHERE r.igoRequestId IS NULL
+SET r.igoRequestId = r.requestId
+REMOVE r.requestId
+RETURN true;
+
+MATCH (r: Request)
+WHERE r.genePanel IS NULL
+SET r.genePanel = r.recipe
+REMOVE r.recipe
+RETURN true;
+
+// sample metadata-level property name changes
+// - cmoSampleClass --> sampleType
+// - specimenType --> sampleClass
+// - oncoTreeCode --> oncotreeCode
+// - recipe --> genePanel
+
+// remove existing sample type data
+MATCH (sm: SampleMetadata)
+REMOVE sm.sampleType
+RETURN true;
+
+MATCH (sm: SampleMetadata)
+WHERE sm.sampleType IS  NULL
```

Comment:
Need to change this query to 

````
MATCH (sm: SampleMetadata)
REMOVE sm.sampleType
RETURN true;

MATCH (sm: SampleMetadata)
WHERE sm.sampleType IS  NULL
SET sm.sampleType = sm.cmoSampleClass
REMOVE sm.cmoSampleClass
RETURN true;

Reply:
it's here already :) see a few lines above